src/fides/api/service/connectors/query_config.py

```diff
@@ -537,6 +547,33 @@ def generate_update_stmt(
         logger.info("query = {}, params = {}", Pii(query_str), Pii(update_value_map))
         return self.format_query_stmt(query_str, update_value_map)
 
+    def get_partition_clauses(
+        self,
+    ) -> List[str]:
+        """
```

Comment:
Since we for sure only support bigquery at the moment, it felt too early to move this to the generic location, but that's just my opinion